resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -173,7 +173,7 @@ function task_install {
     ln -s "../apps/${real_name}/bin/<%= basename %>" "${DESTDIR}${symbindir}/<%= basename %>"
     ln -s "../server/apps/${real_name}/bin/<%= basename %>" "${DESTDIR}${uxbindir}/<%= basename %>"
 <% end -%>
-    install -d -m 0755 "${DESTDIR}${rundir}"
+    install -d -m 0755 -o <%= EZBake::Config[:user] %> -g <%= EZBake::Config[:group] %> "${DESTDIR}${rundir}"
```

Comment:
This is the line that makes me think we at least set rundir ownership at install time to the proper USER/GROUP on RedHat: https://github.com/puppetlabs/ezbake/blob/1.1.1/resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/ezbake.spec.erb#L271.  Not sure if it's possible to do something similar for Debian?